packages/admin-api-client/README.md

```diff
@@ -0,0 +1,243 @@
+| userAgentPrefix?    | `string`         | Any prefix you wish to include in the User-Agent for requests made by the library.                                                                                                                                                                                                                                  |
+| customFetchAPI?     | `CustomFetchAPI` | A custom fetch function that will be used by the client when it interacts with the API. Defaults to the [browser's Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API).                                                           |
+
+## Client properties
+
+| Property      | Type                                                                                                                                                                       | Description                                                                                                                                                                                                                                                                                                                                                                |
+| ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| config        | `ClientConfig`                                                                                                                                                             | Configuration for the client                                                                                                                                                                                                                                                                                                                                               |
+| getHeaders    | `(customHeaders?: Record<string, string>) => Record<string, string>`                                                                                                       | Returns Admin API specific headers needed to interact with the API. If `customHeaders` is provided, the custom headers will be included in the returned headers object.                                                                                                                                                                                               |
```

Comment:
nit: Our TS config discourages us from using `Record<any, any>` so in the types files, we've switched to use `{[key: string]: string}` - that doesn't necessarily need to change in the README but for the other client's, I've updated them to use the format we use in the types file.